### PR TITLE
📝 swagger doc updates for application and app summary

### PR DESCRIPTION
**Pull request by anncatton**
Updates for Application and ApplicationSummary responses. Adds ApplicationBase schema
- fixes some parsing errors
- cleanup of defunct endpoints
- removes `submittedAtUtc` field from `Application` interface as it is included in the `ApplicationBaseFields` interface

### CODE REVIEW

File: src/domain/interface.ts [hunk 1]
```diff
@@ -260,7 +260,6 @@ export interface Sections {
 export interface Application extends ApplicationBaseFields {
   appNumber: number;
   submitterEmail: string;
-  submittedAtUtc: Date;
```
Comment (joneubank) [inline src/domain/interface.ts:263]: why is this deleted in a commit that only mentions swagger updates?

Comment (anncatton) [reply src/domain/interface.ts:263]: because i failed you @joneubank 
pr description is now updated

**Review by joneubank — APPROVED** [review]
